src/schema/paddle.ts

```diff
@@ -82,15 +127,47 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         address: region ? { countryCode: region as CountryCode } : undefined,
       });
 
-      const items = pricePreview?.details?.lineItems.map((item) => ({
-        label: item.price.name,
-        value: item.price.id,
-        price: item.formattedTotals.total,
-        priceUnformatted: Number(item.totals.total),
-        currencyCode: pricePreview?.currencyCode as string,
-        extraLabel: (item.price.customData as GQLCustomData)?.label,
-        appsId: (item.price.customData as GQLCustomData)?.appsId,
-      }));
+      const items = pricePreview?.details?.lineItems.map((item) => {
+        const isOneOff = !item.price?.billingCycle?.interval;
+        const isYearly = item.price?.billingCycle?.interval === 'year';
+        const duration =
+          isOneOff || isYearly
+            ? SubscriptionCycles.Yearly
+            : SubscriptionCycles.Monthly;
+        const priceAmount = getPriceFromPaddleItem(item);
+        const months = duration === SubscriptionCycles.Yearly ? 12 : 1;
+        const monthlyPrice = Number(
```

Comment:
This one as well I guess?

Reply:
As now I've just replicated the client logic since we need to pass the values directly into the client. If we don't want to have this logic here I can simplify returning the items and moving the transformation in our client side.